homeassistant/components/switch/__init__.py

```diff
@@ -30,6 +30,8 @@
 
 ATTR_TODAY_MWH = "today_mwh"
 ATTR_CURRENT_POWER_MWH = "current_power_mwh"
+ATTR_CURRENT_POWER_W = "Current power (W)"
+ATTR_TOTAL_CONSUMPTION_KWH = "Total energy (kWh)"
```

Comment:
Is there then a way to set a friendly name for the attribute ? Otherwise it does not really looks good in the interface: 
<img width="290" alt="screen shot 2016-01-31 at 16 03 04" src="https://cloud.githubusercontent.com/assets/534819/12702871/bedf73e6-c834-11e5-8897-6a53285694a2.png">
<img width="318" alt="screen shot 2016-01-31 at 16 07 12" src="https://cloud.githubusercontent.com/assets/534819/12702874/c3afd172-c834-11e5-9b46-dac4cc4dbdfa.png">

Also, a power should be in [W] not [xWh], and shouldn't power and total energy be common to all switches ?
Moreover, is PROP_TO_ATTR used for something ?